backend/blueprint/example.yaml

```diff
@@ -0,0 +1,17 @@
+runtime: python
```

Comment:
You are using a custom runtime and specifying the run time as python. Isn't it wrong as gcloud will not allow the practice.

Reply:
@yashLadha This is an example yaml file, I pushed. My current yaml specifies runtime as custom

Reply:
As I have pushed the file that I have pushed to gcp by removing config. You haven't done the same. It is looking very wrong to me as you are well aware of the practices

Reply:
@yashLadha I pushed because I want new users to just copy this example file and make `app.yaml` and after making changes they can push to their own gcp account